sdjournal/read.go

```diff
@@ -188,6 +188,29 @@ process:
 	return
 }
 
+func (r *JournalReader) FollowAsync(until <-chan time.Time, recieve chan<- []byte) <-chan error {
+	errCh := make(chan error, 1)
+	go func() {
+		var msg = make([]byte, 64*1<<(10))
+		for {
+			c, err := r.Read(msg)
+			if err != nil && err != io.EOF {
```

Comment:
On thinking about this a bit more, we could probably just generalise the two cases and `Follow` could simply wrap around the async version